guides/version_control_guidance.md

```diff
@@ -11,3 +11,5 @@ Avoid putting everything related to your project in a single repository.
 Put each component (application, package, library, etc) into its own repository, and use [GitHub teams](https://help.github.com/en/articles/organizing-members-into-teams) to manage access to them.
 
 This makes it easy to apply version control to each component individually, whilst also ensuring that access to the repositories is simple to control.
+
+For additional guidance on choosing between a mono repo and a multi repo, see the [guidance](/guides/choosing_mono_or_multi_repo.md).
```

Comment:
I feel this page is a bit contradictory now as it basically says "don't use a monorepo" and then says "for guidance on choosing between..." so I think we should look to firm it up one way or another

Reply:
Yes agreed, I felt that too.  Based on the overall theme of feedback, I've changed to tone to be less around a decision and more multi-repo is our standard, read the background here.

So have dropped the language around "choosing".  Made the case for mono repos to be specific.  ie one project and one team.